packages/mask/src/extension/popups/normal-client.tsx

```diff
@@ -19,7 +18,6 @@ if (
 ) {
     console.time('[SSR] Fill data')
     await Promise.all([
-        activateSiteAdaptorUI(),
```

Comment:
why did you remove this? if you remove this, where do you start the site adaptor?

Reply:
Why do we need to start site adaptor on popups page?

Reply:
oh, we're expected to not start this (we used to start it because some plugin crashes), if there is no bug with this, I'm okay with this.